osbs/http.py

```diff
@@ -34,6 +41,22 @@
 logger = logging.getLogger(__name__)
 
 
+class OSBSRetry(Retry):
+    def is_forced_retry(self, method, status_code, has_retry_after=False):
+        if status_code in HTTP_RETRIES_STATUS_FORCELIST:
+            logger.debug("Server returned returned %s", status_code)
+            return True
+        else:
+            return False
+
+    def increment(self, method=None, url=None, response=None,
```

Comment:
Yep. RHEL7 uses requests 2.6.0, so we can't use a lot of goodies from recent versions